src/actions/__tests__/modalActions.test.js

```diff
@@ -0,0 +1,31 @@
+import mockStore from '../../utils/testUtils';
+
+import {
+  emailSignupAction,
+  formSignupAction,
+  hideModalActions,
+  displayModalActions,
+} from '../modalActions';
+
+describe('test modals logic', () => {
+  const store = mockStore();
+  beforeEach(() => {
+    store.clearActions();
+  });
+
+  test('show modal', async () => {
```

Comment:
You can remove this test file from your branch. I believe @kburudi handled these tests a little bit better.